app/frontend/src/features/messages/groupchats/GroupChatView.tsx

```diff
@@ -123,9 +121,10 @@ export default function GroupChatView() {
               onClick={handleClick}
               aria-label="Menu"
               aria-haspopup="true"
+              aria-controls="more-menu"
               innerRef={menuAnchor}
             >
-              <SettingsIcon />
+              <OverflowMenuIcon />
             </HeaderButton>
             <Menu
               id="simple-menu"
```

Comment:
```suggestion
              id="more-menu"
```
The ID is meant to match with what you got for `aria-controls` for it to be useful/improve accessibility (I think mainly for screen readers).

Reply:
Oops, that's from messing up the rebase